python/requirements/anyscale-requirements.txt

```diff
@@ -0,0 +1,10 @@
+jupyterlab
+ipywidgets
```

Comment:
upgrade `ipywidgets` to `>8.0.6`?

Reply:
discuss offline, the version is constraint by other ray dependencies